src/libcore/macros.rs

```diff
@@ -190,9 +190,18 @@ macro_rules! debug_assert_eq {
 }
 
 /// Helper macro for unwrapping `Result` values while returning early with an
-/// error if the value of the expression is `Err`. Can only be used in
-/// functions that return `Result` because of the early return of `Err` that
-/// it provides.
+/// error if the value of the expression is `Err`. It additionally provides
+/// automatic conversions of errors into other errors using `From`.
+/// Can only be used in functions that return `Result` because of the
+/// early return of `Err` that it provides.
+///
+/// `try!` unwraps the given `Result` and, in case of an `Ok` variant, the
```

Comment:
"unwrap" is an unfortunate choice of words here, since `unwrap()` is a method, and it doesn't use it